packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -384,6 +384,21 @@ export const scheduleTimeout: any =
 export const cancelTimeout: any =
   typeof clearTimeout === 'function' ? clearTimeout : (undefined: any);
 export const noTimeout = -1;
+export const queueMicrotask: (Function => void) | undefined =
+  typeof queueMicrotask === 'function'
+    ? queueMicrotask
+    : typeof Promise !== 'undefined'
+    ? callback =>
+        Promise.resolve(null)
+          .then(callback)
+          .catch(handleErrorInNextTick)
+    : scheduleTimeout;
```

Comment:
Let's leave a TODO here to consider the best fallback. I think we need to warn consumers to use a Promise polyfill.

And then as a maybe as a last resort it should schedule a Scheduler task.

Reply:
Added item to the umbrella

Reply:
Added TODO